lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -189,9 +189,15 @@
                 successMessage: function() {
                     return HtmlUtils.joinHtml(
                         this.indicators.success,
-                        StringUtils.interpolate(
-                            gettext('We\'ve sent a message to {email_address}. Click the link in the message to reset your password.'),  // eslint-disable-line max-len
-                            {'email_address': this.model.get(this.options.emailAttribute)}
+                        HtmlUtils.interpolateHtml(
+                            gettext('We\'ve sent a message to {email}. Click the link in the message to reset your password. Didn\'t receive the message? See {anchorStart}support{anchorEnd}.'),  // eslint-disable-line max-len
```

Comment:
Let's use the same phrase, "contact technical support" as on the other page. "See support" doesn't make much sense to me ("get support" or "contact support" is fine though).